lint/action.yaml

```diff
@@ -12,10 +12,17 @@ runs:
   using: composite
   steps:
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
       if: inputs.checkout-repo == 'true'
       with:
         fetch-depth: 0
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install action-docs
```

Comment:
Si se have to account for yarn here?

Reply:
@tagoro9 I'm using npx now, I forgot about this. 

I'm just ensuring that node is setup and it should be good.